### PR TITLE
Add CSC Global

### PR DESCRIPTION
**Pull request by ares466**

### CODE REVIEW

File: _data/domains.yml
Comment (Carlgo11) [inline _data/domains.yml]:
```suggestion
      url: https://www.cscglobal.com/
```

File: _data/domains.yml
Comment (Carlgo11) [inline _data/domains.yml]:
```suggestion
      img: csc.png
```

File: _data/domains.yml
Comment (Carlgo11) [inline _data/domains.yml:69]: Could you provide any documentation page for how to enable 2FA?

File: _data/domains.yml
Comment (Carlgo11) [inline _data/domains.yml:66]: Is it CSC or CSC Global?

**Review by Carlgo11 — DISMISSED** [review]
How about this image instead?
csc

File: _data/domains.yml
Comment (ares466) [reply _data/domains.yml:69]: Unfortunately, we do not publish any documentation related to 2FA. The instructions are provided to our customers during onboarding.

File: _data/domains.yml
Comment (ares466) [reply _data/domains.yml:66]: Publically we go by CSC. Unfortunately, the csc domain name was owned by the other CSC (computer services corporation) which was bought out by DXC Technology. Long story short - we are CSC, but our website is cscglobal.com.

File: _data/domains.yml
Comment (ares466) [reply _data/domains.yml]: I will run this by our marketing department and raise a new pull request if they are okay with the change. For the time being, will this one suffice?

**Review by ares466 — COMMENTED** [review]
All changes have been made other than the actual logo. Let me know if there is anything else. Thanks!

File: _data/domains.yml
Comment (Carlgo11) [reply _data/domains.yml:69]: In that case I'll need some sort of screenshot of the setup process or login process (with any personal information blurred out).
You can post it here as a comment by dragging and dropping the image to the message field.

Comment (ares466) [reply _data/domains.yml:69]: csc_2factor_login

Let me know if that works...

Comment (brycx) [reply _data/domains.yml:69]: I don't see anything related to `hardware` here. Also, should we have an exception for requiring the Duo app?

Comment (Carlgo11) [reply _data/domains.yml:69]: @brycx no the duo app is still software 😛

Comment (ares466) [reply _data/domains.yml:69]: Hardware is accomplished through key fobs provided by Duo on request. Very few of our customers use hardware key fobs because there is an extra cost.

If a customer has a key fob, they can enter the passcode provided by the key fob via the third option in the image above.

The Duo app is only required for the "Duo Push" option. Neither phone call, SMS, or hardware require the user have the Duo app. That being said, I'll let you guys decide if this should be considered an exception.

Comment (brycx) [reply _data/domains.yml:69]: Given what you've said, the current state sounds reasonable.

**Review by brycx — DISMISSED** [review]

**Comment by Carlgo11** [comment]
I just want to make sure you're okay with the current image you've chosen. It will look like this once merged:
screenshot

If you're fine with that then just tell me and I'll merge the PR 😉

**Comment by ares466** [comment]
I am confirming with Marketing and will respond in the next few days. Thanks for checking - I don't that's quite the look we're going for.

**Comment by ares466** [comment]
I've committed a new logo. Thanks!

**Review by brycx — CHANGES_REQUESTED** [review]
Hi @ares466, there seems to be missing some documentation here. Either add a link to such or take a screenshot with personal information blurred out, that documents the process of setting up 2FA. Thank you.

**Comment by Carlgo11** [comment]
Closing due to inactivity.

**Pull request closed without merging**
